openquake/engine/calculators/hazard/general.py

```diff
@@ -100,9 +100,15 @@ def execute(self):
         distribution, but it can be overridden in subclasses.
         """
         csm = self.composite_model
+        rlzs_assoc = csm.get_rlzs_assoc()
+        # temporary hack
```

Comment:
Please check if you want this temporary hack to be merged or you prefer to work on it before merging